### PR TITLE
address feedback of PR 1128

### PR DESCRIPTION
**Pull request by rupeshparab**
Ref - https://github.com/benetech/MathShare/pull/1128

### CODE REVIEW

**Comment by jscholes** [comment]
@rupeshparab Does this incorporate movement of the back button after the `<h1>`?  I only see the removal of the skip link in the diff, but I could easily be missing something.

**Comment by jscholes** [comment]
@rupeshparab Never mind; going through my email and see you've already asked a question about that in https://github.com/benetech/MathShare/pull/1128#issuecomment-617291921.

**Comment by johnhbenetech** [comment]
@rupeshparab is this good to go?

**Comment by rupeshparab** [comment]
@johnhbenetech there is a minor frontend change in the feedback suggested by James
I have moved 'Back to all problems' button to the right, attaching its screenshot:
image

Will need someone from product team to check if it looks ok

**Comment by johnhbenetech** [comment]
@alexrcabral confirm this looks as intended

File: src/components/Editor/components/ProblemHeader/index.js [hunk 1]
```diff
@@ -158,6 +140,22 @@ export default class ProblemHeader extends Component {
...
                         <span id="math-ellipsis" className={`flex-grow-1 ${problem.mathEllipsis}`}>&nbsp;</span>
                         {exampleLabel}
+                        <Button
+                            id="backBtn"
+                            className={classNames('btn', 'pointer', problem.button)}
+                            additionalStyles={['default']}
+                            type="button"
+                            icon="arrow-left"
+                            onClick={this.props.goBack}
+                            tabIndex="-1"
```
Comment (jscholes) [inline src/components/Editor/components/ProblemHeader/index.js:150]: @rupeshparab Why does the button have a tabindex of "-1"?

Comment (rupeshparab) [reply src/components/Editor/components/ProblemHeader/index.js:150]: not sure why the tabindex was added
it was added in this commit along with screen reader text
https://github.com/benetech/MathShare/commit/a6cad4d0b125285f95d01ff14be15c449d5c1a10

**Comment by jscholes** [comment]
@rupeshparab Okay... I have no idea what that was all about, but it should be removed and then this can be merged.

**Comment by rupeshparab** [comment]
@jscholes 
I have removed the tabindex

**Comment by alexrcabral** [comment]
@johnhbenetech @rupeshparab Yes, this looks as intended

**Comment by johnhbenetech** [comment]
@rupeshparab good to merge

**Pull request merged**